src/pages/trade-types/components/_tabs.tsx

```diff
@@ -4,10 +4,17 @@ import { Flex, Desktop, Mobile } from 'components/containers'
 import { Text } from 'components/elements'
 import device, { SizeType } from 'themes/device'
 import { ReactComponent as Info } from 'images/svg/trade-types/info2.svg'
+import { TString } from 'types/generics'
+import { Localize } from 'components/localization'
+
+export type LocalizeComponentAttributes = {
+    text: TString
+    components?: ReactElement[]
+}
```

Comment:
We can refactor it like this
```suggestion
export type LocalizeComponentAttributes = {
    text: TString
    components?: ReactElement[]
} | TString
```
And we'll not need to write types like this: `LocalizeComponentAttributes | TString`